cluster/store/store.go

```diff
@@ -258,6 +228,46 @@ func (st *Store) Open(ctx context.Context) (err error) {
 	return nil
 }
 
+func (st *Store) init() (logCache *raft.LogCache, err error) {
+	if err = os.MkdirAll(st.raftDir, 0o755); err != nil {
+		return nil, fmt.Errorf("mkdir %s: %w", st.raftDir, err)
+	}
+
+	// log store
+	st.logStore, err = raftbolt.NewBoltStore(filepath.Join(st.raftDir, raftDBName))
+	if err != nil {
+		return nil, fmt.Errorf("bolt db: %w", err)
+	}
+
+	// log cache
+	logCache, err = raft.NewLogCache(logCacheCapacity, st.logStore)
```

Comment:
**suggestion (if-minor):** why not making `logCache` part of the store so we don't need to return it specifically

Reply:
Because we only use if for init and it is not used further